csharp-basics/exercises/FlowOfControl/FlowControl/LargestNumber/Program.cs

```diff
@@ -15,13 +15,28 @@ static void Main(string[] args)
 
             Console.WriteLine("Input the 3rd number: ");
             var input3 = Console.ReadLine();
-        
-            /*
-            todo - expected output:
-            Input the 1st number: 25
-            Input the 2nd number: 78
-            Input the 3rd number: 87
-             */
+
+            if (double.Parse(input1) <= double.Parse(input2))
+            {
```

Comment:
Lai nav vairākas reizes jātaisa parse vienam un tam pašam inputam, saglabā to iepriekš un izmanto visur kur pēc tam vajag